src/zenml/client.py

```diff
@@ -556,11 +556,10 @@ def is_inside_repository(file_path: str) -> bool:
             True if the file is inside the active ZenML repository, False
             otherwise.
         """
-        repo_path = Client.find_repository()
-        if not repo_path:
-            return False
-
-        return repo_path in Path(file_path).resolve().parents
+        if repo_path := Client.find_repository():
+            return repo_path in Path(file_path).resolve().parents
+        else:
+             return False
```

Comment:
No need for else after return :) 
```suggestion
        return False
```

Reply:
okay